src/ssl_asn1.c

```diff
@@ -1019,7 +1019,7 @@ static void wolfssl_asn1_integer_reset_data(WOLFSSL_ASN1_INTEGER* a)
     /* No data, not negative. */
     a->negative = 0;
     /* Set type to positive INTEGER. */
-    a->type = V_ASN1_INTEGER;
+    a->type = WOLFSSL_ASN1_TYPE_INTEGER;
```

Comment:
WOLFSSL_V_ASN1_INTEGER

Reply:
yup better.  fixed globally for all `WOLFSSL_ASN1_TYPE_*`s.